apps/etterlatte-brev-api/.nais/dev.yaml

```diff
@@ -106,7 +106,6 @@ spec:
   accessPolicy:
     outbound:
       rules:
-        - application: etterlatte-pdl-proxy
```

Comment:
Hmm, brev bruker proxy, men mulig denne (outbound rule) aldri er i bruk siden vi går mot dev-fss og external host 🤔

Reply:
Tror det stemmer - går man via external så gjelder ikke outbound

Reply:
Forskjell på proxy og pdl-proxy 😉

Reply:
Men da er neste spørsmål - kan denne flyttes til gcp? :)

Reply:
> Forskjell på proxy og pdl-proxy 😉

Sant! Slo meg ikke at det var `etterlatte-pdl-proxy` siden den er pensjonert for _lenge siden_

Reply:
etterlatte-proxy er vår inngangsport til det vi bruker som køyrer på fss, så den vil vi nok trengje ei god stund framover - ser for meg det tar tid før feks utbetaling er ute.